src/appleseed-max-impl/appleseedrenderer/renderersettings.cpp

```diff
@@ -184,6 +187,18 @@ const char* get_lighting_engine_type(const int lighting_engine_type)
     }
 }
 
+const char* get_lighting_algorithm_type(const int lighting_algorithm_type)
+{
+    switch (lighting_algorithm_type)
+    {
+    case 0:  return "cdf";
```

Comment:
Indent `case` and `default statements with 2 spaces. Check how other `switch` statements are indented.